packages/jest-jasmine1/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "jest-jasmine1",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "src/index.js",
+  "dependencies": {},
+  "scripts": {
+    "test": "jest"
+  },
+  "jest": {
+    "testEnvironment": "<rootDir>/../../src/environments/NodeEnvironment"
```

Comment:
that line is a little sketchy because it goes outside of the package's scope.
without it `typeof window === object // true` and jasmine won't export commonjs module.